internal/resolvers/wikipedia/load_test.go

```diff
@@ -0,0 +1,137 @@
+package wikipedia
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/Chatterino/api/pkg/utils"
+	qt "github.com/frankban/quicktest"
+	"github.com/go-chi/chi/v5"
+)
+
+func testLoadAndUnescape(c *qt.C, locale, page string) (cleanTooltip string) {
+	const urlFormat = "https://%s.wikipedia.org/wiki/%s"
+	urlString := fmt.Sprintf(urlFormat, locale, page)
```

Comment:
Do we have to declare a variable here that's used only in 1 place? Maybe I'm nitpicky.
```suggestion
	urlString := fmt.Sprintf("https://%s.wikipedia.org/wiki/%s", locale, page)
```